_observing-your-data/query-insights/top-n-queries.md

```diff
@@ -96,6 +94,9 @@ GET /_insights/top_queries?type=memory
 ```
 {% include copy-curl.html %}
 
+If you are not seeing results, verify top N is enabled for the target metric type and ensure search requests were sent during the current time window.
```

Comment:
@dzane17 Here's a rewording suggestion. Could you also clarify what "current" means in this context?
```suggestion
If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current time window.
```